Forge/Server/Data/LiteDbCharacterService.cs

```diff
@@ -43,6 +43,26 @@ public bool Update(CharacterModel character)
                 .Update(character);
         }
 
+        public bool DeleteOne(Guid id)
+        {
+            var character = _liteDb.GetCollection<CharacterModel>("Character")
+                .Include(x => x.Tags)
+                .Find(x => x.Id == id)
+                .FirstOrDefault();
+            character.IsDeleted = true;
+            return Update(character);
+        }
+
+        public bool RestoreOne(Guid id)
+        {
+            CharacterModel character = _liteDb.GetCollection<CharacterModel>("Character")
```

Comment:
Please use the "FindOne" repository method instead of directly accessing the collection. Also use "var".